src/bo4e/enum/waehrungscode.py

```diff
@@ -0,0 +1,191 @@
+    MGA = "MGA"  #: Malagasy Ariary
+    MKD = "MKD"  #: Denar
+    MMK = "MMK"  #: Kyat
+    MNT = "MNT"  #: Tugrik
+    MOP = "MOP"  #: Pataca
+    MRO = "MRO"  #: Ouguiya
+    MUR = "MUR"  #: Mauritius Rupee
+    MVR = "MVR"  #: Rufiyaa
+    MWK = "MWK"  #: Kwacha
+    MXN = "MXN"  #: Mexican Peso
+    MXV = "MXV"  #: Mexican Unidad de Inversion (UDI)
+    MYR = "MYR"  #: Malaysian Ringgit
+    MZN = "MZN"  #: Mozambique Metical
+    NAD = "NAD"  #: Namibia Dollar
+    NGN = "NGN"  #: Naira
+    NIO = "NIO"  #: Cordoba Oro
+    NOK = "NOK"  #: Norwegian Krone
+    NPR = "NPR"  #: Nepalese Rupee
+    NZD = "NZD"  #: New Zealand Dollar
+    OMR = "OMR"  #: Rial Omani
+    PAB = "PAB"  #: Balboa
+    PEN = "PEN"  #: Nuevo Sol
+    PGK = "PGK"  #: Kina
+    PHP = "PHP"  #: Philippine Peso
+    PKR = "PKR"  #: Pakistan Rupee
+    PLN = "PLN"  #: Zloty
+    PYG = "PYG"  #: Guarani
+    QAR = "QAR"  #: Qatari Rial
+    RON = "RON"  #: New Romanian Leu
+    RSD = "RSD"  #: Serbian Dinar
+    RUB = "RUB"  #: Russian Ruble
+    RUR = "RUR"  #: Russian Ruble
+    RWF = "RWF"  #: Rwanda Franc
+    SAR = "SAR"  #: Saudi Riyal
+    SBD = "SBD"  #: Solomon Islands Dollar
+    SCR = "SCR"  #: Seychelles Rupee
+    SDG = "SDG"  #: Sudanese Pound
+    SEK = "SEK"  #: Swedish Krona
+    SGD = "SGD"  #: Singapore Dollar
+    SHP = "SHP"  #: Saint Helena Pound
+    SLL = "SLL"  #: Leone
+    SOS = "SOS"  #: Somali Shilling
+    SRD = "SRD"  #: Surinam Dollar
+    SSP = "SSP"  #: South Sudanese Pound
+    STD = "STD"  #: Dobra
+    SVC = "SVC"  #: El Salvador Colon
+    SYP = "SYP"  #: Syrian Pound
+    SZL = "SZL"  #: Lilangeni
+    THB = "THB"  #: Baht
+    TJS = "TJS"  #: Somoni
+    TMT = "TMT"  #: Turkmenistan New Manat
+    TND = "TND"  #: Tunisian Dinar
+    TOP = "TOP"  #: PaÊ»anga
+    TRY = "TRY"  #: Turkish Lira
+    TTD = "TTD"  #: Trinidad and Tobago Dollar
+    TWD = "TWD"  #: New Taiwan Dollar
+    TZS = "TZS"  #: Tanzanian Shilling
+    UAH = "UAH"  #: Hryvnia
+    UGX = "UGX"  #: Uganda Shilling
+    USD = "USD"  #: US Dollar
+    USN = "USN"  #: US Dollar (Next day)
+    USS = "USS"  #: US Dollar (Same day)
+    UYI = "UYI"  #: Uruguay Peso en Unidades Indexadas (URUIURUI)
+    UYU = "UYU"  #: Peso Uruguayo
+    UZS = "UZS"  #: Uzbekistan Sum
+    VEF = "VEF"  #: Bolivar
+    VND = "VND"  #: Dong
+    VUV = "VUV"  #: Vatu
+    WST = "WST"  #: Tala
+    XAF = "XAF"  #: CFA Franc BEAC
+    XAG = "XAG"  #: Silver
+    XAU = "XAU"  #: Gold
+    XBA = "XBA"  #: Bond Markets Unit European Composite Unit (EURCO)
+    XBB = "XBB"  #: Bond Markets Unit European Monetary Unit (E.M.U.-6)
+    XBC = "XBC"  #: Bond Markets Unit European Unit of Account 9 (E.U.A.-9)
+    XBD = "XBD"  #: Bond Markets Unit European Unit of Account 17 (E.U.A.-17)
+    XCD = "XCD"  #: East Caribbean Dollar
+    XDR = "XDR"  #: SDR (Special Drawing Right)
+    XOF = "XOF"  #: CFA Franc BCEAO
+    XPD = "XPD"  #: Palladium
+    XPF = "XPF"  #: CFP Franc
+    XPT = "XPT"  #: Platinum
```

Comment:
Nice I can pay with Palladium or Platinum?!

Reply:
😀 yagni

das bestärkt mich in #126